src/main/target/STM32H725/target.mk

```diff
@@ -0,0 +1,8 @@
+TARGET_MCU        := STM32H725xx
+TARGET_MCU_FAMILY := STM32H7
+
+H723xG_TARGETS += $(TARGET)
```

Comment:
Maintainers: you may want to consider this carefully as this H735/H725 board is now in the list of H723 targets...

Another PR might be in order to add `H7x5xG_TARGETS`.

Reply:
This line can be removed. I don't believe we use that _TARGETS anywhere anymore... I thought I got rid of that.